unitTests/ui/ResultsPerPageTest.ts

```diff
@@ -213,6 +213,19 @@ export function ResultsPerPageTest() {
         });
         expect(test.env.queryController.options.resultsPerPage).toBe(firstChoice);
       });
+
+      it('initialChoice set to non default value is use when default value is in choicesDisplayed', () => {
+        let initialChoice = 5;
```

Comment:
It's a good practice to always use `const` when the variable value is not modified.